cmd/context/context.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package context
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	okContext "github.com/okteto/okteto/pkg/cmd/context"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+type ContextOptions struct {
+	Token           string
+	isOktetoCluster bool
+}
+
+// Context points okteto to a cluster.
+func Context() *cobra.Command {
+	ctxOptions := &ContextOptions{}
+	cmd := &cobra.Command{
+		Use:     "context [url|k8s-context]",
+		Aliases: []string{"ctx"},
+		Args:    utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#context"),
+		Short:   "Points okteto to a cluster",
+		Long: `Points okteto to a cluster
+
+Run
+    $ okteto context
```

Comment:
`login` is a very common action in CLIs. pulumi, aws, docker, heroku, etc have a login command. I personally find very confusing that we're mixing 'login to your okteto cluster'  with the well-known concept of 'use a kubernetes context' for the operation. 

Have we considered using a different term that doesn't already have a meaning in the world of kubernetes?

Reply:
maybe login still exists as a command (only usable in okteto clusters), and it sets the context to the one that we create?  That way,  developers who don't use okteto cloud/enterprise (which is like a third of our user base I think), can still use this to set the kubernetes context?

Reply:
another option could be to make this only work for okteto clusters. if you're on a vanilla kube cluster, you use standard  tooling for this (e.g. kubie, kubectl config set-conext, etc..)

Reply:
This method comes to set okteto to always point to the same k8s context allowing you to have another one used for your production cluster. Using `kubectl config set-conext` will set their context on their local kubeconfig but not on okteto

Reply:
I think that making the CLI work the same way in an Okteto Instance or your own cluster is key here. Otherwise, the developer will have to think: Am I using an Okteto instance? Then I have to do `login` too, but when I work with my own cluster I don't... 

The idea behind this is that the developer only has to type **one command** to point to the desired cluster, no matter if it is their own or an Okteto instance.

Reply:
> I think that making the CLI work the same way in an Okteto Instance or your own cluster is key here

Reply:
> the developer will have to think: Am I using an Okteto instance?

Running `okteto context $URL` still involves that decision no?  You have to know you're on okteto cloud/enterprise AND the URL to provide. 

One thing to consider is that standard for kubernetes tooling directly works with your current kubernetes context without you requiring any extra commands (helm and kubectl works like that). So non-cloud users need to run two commands now (okteto context + okteto up) instead of just one.  This might confuse the non-cloud users.